cmake/parthenonConfig.cmake.in

```diff
@@ -0,0 +1,36 @@
+#=========================================================================================
+# (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+#=========================================================================================
+
+# Favor using the kokkos package that was built with parthenon, if one has not been specified
+if (@PARTHENON_EXTERNAL_KOKKOS@)
+  find_package(Kokkos 3 REQUIRED PATHS @Kokkos_DIR@ NO_DEFAULT_PATH)
+else()
+  find_package(Kokkos 3 REQUIRED PATHS @CMAKE_INSTALL_PREFIX@ NO_DEFAULT_PATH)
+endif()
+
+set(Kokkos_BUILT_WITH_CUDA @Kokkos_ENABLE_CUDA@)
+if(${Kokkos_BUILT_WITH_CUDA})
+  string(FIND ${CMAKE_CXX_COMPILER} "nvcc_wrapper" FOUND_WRAPPER)
+  if( ${FOUND_WRAPPER} EQUAL "-1" )
+    if (NOT @EXTERNAL_KOKKOS@)
```

Comment:
Where does this path variable come from is it set somewhere?

Reply:
Changed this to `@PARTHENON_EXTERNAL_KOKKOS@` (should have grep'd for instances of the old variable `EXTERNAL_KOKKOS` sooner)